web/client/themes/default/less/widget.less

```diff
@@ -154,6 +154,57 @@
             margin-left: 5%;
             font-style: italic;
         }
+
+        .widget-selector {
+            flex-grow: 1;
+            position: relative;
+            display: flex;
+            justify-content: center;
+
+            .Select {
+                flex-grow: 1;
+                .Select-control {
+                    margin: 0 auto;
+                    max-width: 60%;
+                    height: 28px;
+
+                    .Select-placeholder {
+                        line-height: 24px;
+                    }
+
+                    .Select-value {
+                        padding-right: 18px;
+                        padding-left: 5px;
+                        line-height: 24px !important;
+
+                        .Select-value-label {
+                            line-height: 24px;
+                        }
+                    }
+                    .Select-input {
+                        height: 24px;
+                    }
+                }
+                .Select-menu-outer {
+                    margin-top: 2px;
+                }
+            }
+
+            .previous-widget, .next-widget {
+                position: relative;
+                z-index: 10;
+            }
+
+            .previous-widget {
+                left: calc(25% - 15px);
+                transform: translateX(-50%);
+            }
+            .next-widget {
+                right: calc(25% - 15px);
+                transform: translateX(+50%);
+            }
```

Comment:
Why do we need to move this two buttons? I think it will be better to let them follow the flex layout default behavior

Reply:
This is done to make it's possible to have select list dropdown as wide as its possible without breaking its default behavior. If I limit select width, internal menu element is limited by this width, making dropdown too narrow to include long words without a breaks.
Arrows with transform:
![Screenshot_31](https://user-images.githubusercontent.com/113525996/193060550-4ef5fd2c-133e-4b95-ba8c-b1707f57a574.jpg)

Without transform:
![Screenshot_32](https://user-images.githubusercontent.com/113525996/193060572-d46d0357-6799-4dcb-b8a7-074a750d6373.jpg)

TLDR; to free up some space for dropdown menu